### PR TITLE
F-1: Initial client project

### PR DESCRIPTION
**Pull request by DamjanPantic**
Initial client project

### CODE REVIEW

File: carrr-client/src/components/NavBar.vue [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<template>
+  <b-navbar type="dark" variant="dark">
+    <b-container>
+      <router-link to="/" tag="b-navbar-brand">CarRR</router-link>
```
Comment (coperope) [inline carrr-client/src/components/NavBar.vue:4]: We should put the full app name here. (?)

File: carrr-client/src/views/About.vue [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<template>
+  <b-container class="mt-4">
+    <b-jumbotron bg-variant="info" text-variant="white" border-variant="dark">
+      <template v-slot:header>Developers</template>
+
+      <hr class="my-4" />
+      <ul>
+        <li>Petar Basić</li>
```
Comment (coperope) [inline carrr-client/src/views/About.vue:8]: My last name is wrong!

File: carrr-client/src/components/NavBar.vue (hunk 1 above)
Comment (DamjanPantic) [reply carrr-client/src/components/NavBar.vue:4]: It should be logo

File: carrr-client/src/views/About.vue (hunk 2 above)
Comment (DamjanPantic) [reply carrr-client/src/views/About.vue]: Sorry teammate

**Pull request merged**